firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/FunctionCallingConfig.kt

```diff
@@ -25,10 +27,14 @@ package com.google.firebase.vertexai.type
  * should match [FunctionDeclaration.name]. With [Mode.ANY], model will predict a function call from
  * the set of function names provided.
  */
-class FunctionCallingConfig(val mode: Mode, val allowedFunctionNames: List<String>? = null) {
+class FunctionCallingConfig
+internal constructor(
+  internal val mode: Mode,
+  internal val allowedFunctionNames: List<String>? = null
+) {
 
   /** Configuration for dictating when the model should call the attached function. */
-  enum class Mode {
+  internal enum class Mode {
```

Comment:
Do other libraries have this hidden too?

Reply:
Yes, it has been changed in other platforms too